collectors/carbonblack/README.md

```diff
@@ -8,23 +8,45 @@ forward logs to the Alert Logic CloudInsight backend services.
 
 # Installation
 
-Refer to [CF template readme](./cfn/README.md) for installation instructions.
+### 1. How to obtain an API Secret Key and API ID
+
+[Authentication](https://developer.carbonblack.com/reference/carbon-black-cloud/authentication/)
+
+1. Log into your Carbon Black Cloud Organization.
+2. Navigate to Settings > API Keys.
+3. Click “Add API Key”.
+4. Configure Name, Access Level, etc.
+5. Obtain your API Secret Key and API ID pair.
+
+[AuditLogEvents](https://developer.carbonblack.com/reference/carbon-black-cloud/cb-defense/latest/rest-api/)
+For Audit Log Events API use access level - API
+
+![ScreenShot](./docs/carbonblack_credentials_api.png)
+
+[SearchRequest](https://developer.carbonblack.com/reference/carbon-black-cloud/platform/latest/alerts-api/)
+For all Search Request API use access level (custom) and custom access level (View All)
```

Comment:
Could you explain that the second `SearchRequest` access level is used for collecting Alerts?